### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,289 @@
...
+  - the development environment,
+  - the list of the dependencies and their versions.
+
+* [README BSP](./README_BSP.md) recommended for users familiar with the
+  <BOARD_MANUFACTURER> SDK/BSP and advanced usage on how to customize the build
+  process.
+
+## Board Technical Specifications
+
+|                         |                                 |
```
[the remark this answers is not in the document]
Comment (tdelhomenie-microej) [reply README.md:28]: That does not seem possible easily... As a workaround you can create the table in HTML directly, or use CSS to hide some cells.

[the remark this answers is not in the document]
Comment (frivieremicroej) [reply README.md:28]: Add header : Property / Value

**Review by jcourcelmicroej — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,3 +1,297 @@
...
+|                 |                         |
+| --------------- | ----------------------- |
+| Baudrate        | <Baudrate value>        |
+| Data bits bits  | <Data bits bits value>  |
+| Parity bits     | <Parity bits value>     |
+| Stop bits       | <Stop bits value>       |
+| Flow control    | <Flow control value>    |
+
+If flashed, the pre-compiled application outputs `Hello World` on
+the UART.
```
Comment (jcourcelmicroej) [inline README.md:197]:
```suggestion
```

Comment (jcourcelmicroej) [reply README.md:197]: No pre-compiled application provided.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,276 @@
...
+by selecting the root folder of the cloned repository.
+
+The project contains the following subprojects:
+
+- `vee-port`: Contains the <VEE_PORT> configuration description. 
+  Some modules are described in a specific sub-folder / with some optional configuration files
+  (`.properties` and / or `.xml`).
+
+- `bsp`: Contains a ready-to-use BSP software project for the <BOARD_NAME> board, including a <CIDE> project, 
+  an implementation of MicroEJ core engine (and extensions) port on <RTOS> and the <BOARD_NAME> board support package.
```
Comment (jcourcelmicroej) [inline README.md:224]:
```suggestion
  an implementation of MicroEJ Core Engine (and extensions) port on <RTOS> and the <BOARD_NAME> board support package.
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,276 @@
...
+<VEE_PORT> (Please refer to the [RELEASE NOTES](./RELEASE_NOTES.md) limitations section for more details).
+
+### <VEE_PORT> Build
+
+To build the <VEE_PORT>, use one of these 2 solutions:
+
+- Run the `app` sample Application [on the Simulator](https://docs.microej.com/en/latest/SDK6UserGuide/runOnSimulator.html) 
+  or [on Device](https://docs.microej.com/en/latest/SDK6UserGuide/runOnDevice.html). 
+  This automatically builds the <VEE_PORT> and uses it.
+- Launch the `buildVeePort` task on the `vee-port` project.
```
Comment (jcourcelmicroej) [inline README.md:247]:
```suggestion
- Launch the `buildVeePort` Gradle task on the `vee-port` project. 
```

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tdelhomenie-microej) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (tdelhomenie-microej) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (tdelhomenie-microej) [reply README.md]: Done

**Pull request merged**